src/main/java/frc/robot/commands/intake/IndexOneCargo.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants;
+import frc.robot.subsystems.IndexerSubsystem;
+
+/**
+ * Run the indexer's gate wheel until a cargo has been indexed, then
+ */
+public class IndexOneCargo extends ConditionalCommand {
+    public IndexOneCargo(IndexerSubsystem indexer) {
+        super(
+            new WaitCommand(0), // Don't do anything if the indexer already has a cargo
+            new SequentialCommandGroup(
+                new InstantCommand(() -> { // Turn on the indexer
+                    indexer.activateConveyor();
+                    indexer.activateGateWheelForIndexing();
```

Comment:
See above, on my first read, I thought activate just mean prepping the system to run but never saw when the motors were actually told to run. Uses terminology like "startConveyor" or "runConveyor" or something similar to be more explicit to what is happening

Reply:
Noted. I changed "deactivate" to "stop" at one point but hadn't considered that "activate" could also be ambiguous.